src/ShowContactForm.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
```

Comment:
I like where you're going with this component, though I think it got a little muddled, you have your open button in here, but then you moved the close button down in to the `ContactForm` and it is calling `this.handleOpen` which doesn't exist in that component, and should actually probably some event that handles closing any way.

I think if you are going to make this some what of a higher order component, you want to consolidate all of that logic in one place, so your close button should be included in this component as well.

But this is a nice attempt, I like seeing the use of composition!